src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/views/EntityDetail/detail.html.twig

```diff
@@ -78,24 +78,9 @@
-        {% include '@Dashboard/EntityDetail/detailAttributeField.html.twig' with {label: ('entity.detail.attribute.' ~ type ~ '.organization_type')|trans, value: entity.organizationTypeAttribute, informationPopup: 'entity.edit.information.' ~ type ~ '.organizationTypeAttribute'} %}
-        {% include '@Dashboard/EntityDetail/detailAttributeField.html.twig' with {label: ('entity.detail.attribute.' ~ type ~ '.organization_unit_name')|trans, value: entity.organizationUnitNameAttribute, informationPopup: 'entity.edit.information.' ~ type ~ '.organizationUnitAttribute'} %}
-        {% include '@Dashboard/EntityDetail/detailAttributeField.html.twig' with {label: ('entity.detail.attribute.' ~ type ~ '.affiliation')|trans, value: entity.affiliationAttribute, informationPopup: 'entity.edit.information.' ~ type ~ '.affiliationAttribute'} %}
-        {% include '@Dashboard/EntityDetail/detailAttributeField.html.twig' with {label: ('entity.detail.attribute.' ~ type ~ '.entitlement')|trans, value: entity.entitlementAttribute, informationPopup: 'entity.edit.information.' ~ type ~ '.entitlementAttribute'} %}
-        {% include '@Dashboard/EntityDetail/detailAttributeField.html.twig' with {label: ('entity.detail.attribute.' ~ type ~ '.principle_name')|trans, value: entity.principleNameAttribute, informationPopup: 'entity.edit.information.' ~ type ~ '.principleNameAttribute'} %}
-        {% include '@Dashboard/EntityDetail/detailAttributeField.html.twig' with {label: ('entity.detail.attribute.' ~ type ~ '.uid')|trans, value: entity.uidAttribute, informationPopup: 'entity.edit.information.' ~ type ~ '.uidAttribute'} %}
-        {% include '@Dashboard/EntityDetail/detailAttributeField.html.twig' with {label: ('entity.detail.attribute.' ~ type ~ '.prefered_language')|trans, value: entity.preferredLanguageAttribute, informationPopup: 'entity.edit.information.' ~ type ~ '.preferredLanguageAttribute'} %}
-        {% include '@Dashboard/EntityDetail/detailAttributeField.html.twig' with {label: ('entity.detail.attribute.' ~ type ~ '.personal_code')|trans, value: entity.personalCodeAttribute, informationPopup: 'entity.edit.information.' ~ type ~ '.personalCodeAttribute'} %}
-        {% include '@Dashboard/EntityDetail/detailAttributeField.html.twig' with {label: ('entity.detail.attribute.' ~ type ~ '.scoped_affiliation')|trans, value: entity.scopedAffiliationAttribute, informationPopup: 'entity.edit.information.' ~ type ~ '.scopedAffiliationAttribute'} %}
-        {% if entity.protocol == "saml20" %}
-            {% include '@Dashboard/EntityDetail/detailAttributeField.html.twig' with {label: ('entity.detail.attribute.' ~ type ~ '.edu_person_targeted_id')|trans, value: entity.eduPersonTargetedIDAttribute, informationPopup: 'entity.edit.information.' ~ type ~ '.eduPersonTargetedIDAttribute'} %}
-        {% endif %}
```

Comment:
Boing!! @Shaky212 

This is one-to-one related to the issue I'm about to fix. Where the Oidcng entity should NOT have the EPTI attribute on the form.

Reply:
I addressed this bug in [0c9d896](https://github.com/SURFnet/sp-dashboard/pull/534/commits/0c9d89658e35f83330155cd8cd00ff1b7ba4b1c2)